app/javascript/review.js

```diff
@@ -39,15 +52,18 @@ route(/\/review\/[\w-]+\/?\d*$/i, async () => {
 });
 
 route(/\/review\/untagged-domains(\/\d*)?/, () => {
-  $(document).on('ajax:success', '.review-add-domain-tag', (e, data) => {
-    const $noTags = $('.no-tags');
-    if ($noTags.length > 0) {
-      $noTags.remove();
-      $('.domain-tag-list p').html('Tagged with: ' + data);
-    }
-    else {
-      $('.domain-tag-list p').append(data);
-    }
-    $(e.target).find('select').selectpicker('val', null);
+  $(document).on('ajax:success', '.review-add-domain-tag', () => {
+    const taggifyInput = document.getElementById('tag_name');
```

Comment:
We still have jQuery :)

`$('#tag_name')`

Reply:
I kinda like sticking to vanilla JS, but fine.

Reply:
I have nothing against vanilla JS, but it's about consistency - we should use either jQuery, or vanilla, but not both if not absolutely necessary.